src/Tribe/Assets.php

```diff
@@ -327,19 +327,24 @@ public static function maybe_get_min_file( $url ) {
 		// Strip the plugin URL and make this relative.
 		$relative_location = str_replace( $base_url, '', $url );
 
+		if ( defined( 'SCRIPT_DEBUG' ) && true === SCRIPT_DEBUG ) {
```

Comment:
We should use 
```suggestion
		if ( defined( 'SCRIPT_DEBUG' ) &&  SCRIPT_DEBUG ) {
```

In case someone set other value like `1`  in `define( 'SCRIPT_DEBUG', 1 );`